### DISCUSSION TITLE
Pre Form Chat not working? Start new conversation.

### DISCUSSION THREAD

**Discussion by JakubSiewniak** [chatwoot/chatwoot#3338] [Q&A] [answered]
Hi. I have updated my inbox to have a pre form chat but it doesn't seem to work (unless I am doing something wrong). Do i need to update client code? widget?

I have updated Pre Form Chat settings as per documentation. But when I open a widget, it doesn't give me a pre chat message or a button to start a conversation, it goes straight to chat.

The main reason for doing this is that it looks like the Start Conversation button will start a new conversation rather then continuing the old one.

My end goal is:
-When user starts a conversation and then comes back to the site, it will continue the same conversation.
-When user starts a conversation and then comes back to the site but conversation has been resolved by the agent, it will trigger a new conversation

Any help appreciated. Thanks

![image](https://user-images.githubusercontent.com/25434450/140752588-0b13b00c-5bf9-401b-ba33-baaf632dcabd.png)

**Comment by muhsin-k** [comment]
@JakubSiewniak Could you please share a screencast for this issue?

**Reply by JakubSiewniak** [reply]
I will get try to get something recorded tomorrow. Anything in particular you want to look at? The basic issue is that when pre-form is enabled or disabled, I always get a standard widget that takes you straight to a conversation.

![image](https://user-images.githubusercontent.com/25434450/140758403-3fc0be3e-cc71-4775-9986-dcb93dc87155.png)

**Comment by muhsin-k** [comment, marked as answer]
Can you please check you are using [`setUser`](https://www.chatwoot.com/docs/product/channels/live-chat/sdk/setup#set-the-user-in-the-widget.) method or not . The pre-chat form will not show if setUser is already invoked

**Reply by JakubSiewniak** [reply]
This was it. So i think that pre form is not what I need then unfortunately. All the users using chat are logged in. Is there any way currently to start a new conversation from widget or close conversation when resolved, so when the user comes back to the site it does not contain the history of a previous conversation? I guess it is being worked on here #2355?

Thanks for a quick response.

**Reply by muhsin-k** [reply]
Yes. Multiple conversations handle this use-case
